pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -604,6 +635,9 @@ def tk_to_qiskit(
         qcirc.add_register(qis_reg)
     symb_map = {Parameter(str(s)): s for s in tkc.free_symbols()}
     range_preds: Dict[Bit, Tuple[List["UnitID"], int]] = dict()
+
+    supported_gate_rebase.apply(tkc)
```

Comment:
Done.

It seems like with the way things were initially `tkc` was a reference  rather than a copy of `tkcirc`.
To make sure its a local copy I've replaced `tkc = tkcirc` with `tkc = tkcirc.copy()`.

An example which helped clarify things for me...

```
def change_circuit(circ: Circuit) -> (Circuit, Circuit):
    copy_circ = circ
    
    SynthesiseTket().apply(copy_circ)
    
    return (copy_circ, circ)
 ```
 With if I apply this function to a `Circuit` then both `copy_circ` and `circ` are changed by the `SynthesiseTket` pass. The two returned circuits are equivalent. This is because `Circuit`s in pytket are mutable.